ui/apps/platform/src/Containers/Administration/Events/AdministrationEventHint.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { ReactElement } from 'react';
+import { List, ListItem, Text } from '@patternfly/react-core';
+
+const listItemRegExp = /^- /;
+
+type LineOrList = string | string[];
+
+function splitHint(hint: string): LineOrList[] {
```

Comment:
Nit: it might be worth a brief comment here describing the intent of this helper function. Although the code is clear I did have to read through it carefully to understand the intended effect.